shared/torngit/github.py

```diff
@@ -47,11 +50,46 @@
         }
     }
 }
-"""
+""",
+        REPOS_FROM_NODEIDS="""
```

Comment:
does github let you fetch the whole list without pagination or will it paginate for you with some default page size?

https://github.com/codecov/engineering-team/issues/139 has an example of pagination with github's graphql api if we need to add it

Reply:
That's a very good question that I probably should have put in the PR description.
I spent quite some time trying to get pagination in this query. From the [docs](https://github.com/codecov/codecov-cli/pull/373) it indicates you can specify a `first`, `last` and get `pageInfo` cursors.

But from the explorer docs the `nodes` query doesn't include the inputs for `first` and `last` (I tried, didn't work) AND the `Repostory` object doesn't include `pageInfo` as a possible field to query. For that reason I don't _think_ there's pagination? At least I couldn't find a way to paginate the request.


![Screenshot 2024-02-07 at 09 08 46](https://github.com/codecov/shared/assets/99758426/a216bd55-471c-4d0d-a170-9dcfdf4d8f76)

Reply:
update: I decided to implement simple pagination from our side just in case.